tests/acceptance/core/testcase/daemonset.go

```diff
@@ -0,0 +1,22 @@
+package testcase
```

Comment:
What @ShylajaDevadiga asked is the preferred way to do it from a Go perspective. There's no need to make separate files or directories since file and directory structure is a biproduct of the package api.